### ISSUE TITLE
[classification] LR warmup for large batch training

### ISSUE THREAD

**Issue by bitfort** [mlcommons/training_policies#54]
Copy a request to modify the benchmark definition:
https://github.com/mlperf/reference/issues/94

There should be a LR warmup method implemented which would make stability and convergence better for larger batch sizes.

For the first couple (5) of epochs learning rate for epoch e shoud equal lr = base_lr * e / 5.
This is a very small change and doesn't break convergence with small batch sizes, but stabilizes training for batch sizes > 256.

**Comment by bitfort** [comment]
SWG:

Will be covered by a the Special Topics Working Group. This may be revisited by SWG after Special Topics provides additional guidance.

**Comment by petermattson** [comment]
We have a hyperparameter policy that will be added to the rules.
